transaction.go

```diff
@@ -43,6 +43,8 @@ type Transaction struct {
 	Customer                     *Customer                 `xml:"customer,omitempty"`
 	BillingAddress               *Address                  `xml:"billing,omitempty"`
 	ShippingAddress              *Address                  `xml:"shipping,omitempty"`
+	TaxAmount                    *NullDecimal              `xml:"tax-amount,omitempty"`
```

Comment:
This could probably simply be a `NullDecimal` rather than a `*NullDecimal`. It makes little difference since it will always be instantiated. 💭